src/main/java/com/amazon/opendistroforelasticsearch/ad/stats/StatNames.java

```diff
@@ -39,13 +39,14 @@
     AD_HC_EXECUTE_REQUEST_COUNT("ad_hc_execute_request_count"),
     AD_HC_EXECUTE_FAIL_COUNT("ad_hc_execute_failure_count"),
     DETECTOR_COUNT("detector_count"),
+    SINGLE_ENTITY_DETECTOR_COUNT("single_entity_detector_count"),
+    MULTI_ENTITY_DETECTOR_COUNT("multi_entity_detector_count"),
```

Comment:
please use single stream and hc detector to be consistent.

Reply:
Prefer to create a separate PR to rename all places to keep it easier to review. Create an issue to track this.